docker-compose-ci.yml

```diff
@@ -0,0 +1,211 @@
+#
```

Comment:
Could you please clarify the difference between docker-compose and docker-compose-ci? Also, is it possible to make one as an extension of the other instead of copy-pasta?

Reply:
Oh, man... this has been a nightmare to figure out. `docker-compose.yml` defines local volumes, so that the data is persisted in your disk. This way when you spin it down and bring it back up again the data is still there. It also defines a "symlink" volume for the DJ repo:

```yaml
dj:
  volumes:
    - .:/code  # map current directory to /code
```

For some reason this doesn't work in CI, so I created a separate file without these volumes. Now it works, but the integration tests are still failing and I don't know why.

I'm going to refactor this whole PR and split it into 2-3, for now I just want to get the integration test passing.

Reply:
@agorajek I split the PR into 4, and still didn't get CI integration tests to work. Going to give up for now, since we can run integration tests manually at least.